client/src/components/App/index.js

```diff
@@ -85,27 +108,20 @@ export default class App extends React.Component {
       '/submit',
       '/myprofile',
     ];
+    const staticRoutes = ['/', '/login', '/signup'];
     return (
       <div>
-        <AppBar logoutHandler={this.logoutHandler} auth={user || admin} />
+        <AppBar
+          logoutHandler={this.logoutHandler}
+          auth={user || admin}
```

Comment:
applicant part only for users not admins. admin won't enter any application

Reply:
exactly, just I thought we used the app bar in admin routes

Reply:
no problem, its just a quick note